Introduction/README.md

```diff
@@ -385,60 +434,40 @@ by the Docker container `fndemouser/gofn:0.0.2` which may be invoked via the
 specified URL.  Now that we've confirmed deployment was successsful, let's
 call our function.
 
-## Calling Your Deployed Function
-
-There are two ways to invoke your deployed function.  The first is using
-the `fn` CLI which makes invoking your function relatively easy.  Type
-the following:
-
-![User Input Icon](images/userinput.png)
->```sh
-> fn invoke goapp gofn
->```
-
-which results in our familiar output message.
-
-```sh
-{"message":"Hello World"}
-```
-
-Of course this is unchanged from when you ran the function locally.
-However when you called "goapp gofn" the fn server looked up the
-"goapp" application and then looked for the Docker container image
-bound to the "gofn" function.
+### Invoke with Curl
 
-The other way to call your function is via HTTP.  The Fn server exposes our
-deployed function at "http://localhost:8080/t/goapp/gofn-trigger", a URL that
-incorporates our application and function trigger as path elements.
+The other way to invoke your function is via HTTP.  The Fn server exposes our
+deployed function at `http://localhost:8080/t/goapp/gofn-trigger`, a URL
+that incorporates our application and function trigger as path elements.
 
 Use curl to invoke the function:
```

Comment:
curl should be in backquotes, e.g. `curl`

Reply:
Fixed.